src/app/games/hive/tests/hive.component.spec.ts

```diff
@@ -138,12 +141,16 @@ describe('HiveComponent', () => {
                 await testUtils.setupState(state);
 
                 // When performing a drop move
-                await testUtils.expectClickSuccess('#remainingPiece_QueenBee_PLAYER_ZERO');
+                await testUtils.expectClickSuccess('.remaining-piece-QueenBee_PLAYER_ZERO');
                 const move: HiveMove = HiveMove.drop(Q, new Coord(0, 0));
-                await testUtils.expectMoveSuccess('#space_0_0', move);
+                await testUtils.expectMoveSuccess('#space-0-0', move);
 
                 // Then the last move should be shown
-                testUtils.expectElementToHaveClass('#stroke_0_0', 'last-move-stroke');
+                testUtils.expectElementToHaveClass('.piece-stroke-0-0', 'last-move-stroke');
```

Comment:
eeeh expect class to have class, big no no

Reply:
fixé since then